src/transformers/bitsandbytes_utils.py

```diff
@@ -0,0 +1,149 @@
+from contextlib import contextmanager
+
+from .utils import is_bitsandbytes_available
+
+
+if is_bitsandbytes_available():
+    import torch
+    import torch.nn as nn
+
+    import bitsandbytes as bnb
+
+
+def set_module_8bit_tensor_to_device(module, tensor_name, device, value=None):
+    """
+    Args:
```

Comment:
This should be before the args are introduced.

Reply:
Should be fixed in 11555491784bf4912aadbce7c9cf641f0c3bd9b9